### PR TITLE
#184 converted classic pipeline to yaml AWS- Templates - AWS CI 3.0

### PR DESCRIPTION
**Pull request by hirendra-singh**
[https://github.com/robe070/cookbooks/issues/184]

converted classic pipeline to yaml pipeline, details are below.

**classic pipeline**
repository: lansa/aws-templates
branch: support/scalable
pipeline name - Templates - AWS CI

**converted classic to yaml pipeline**
repository: hirendra-singh/aws-templates
branch: feature/hiren-aws-yaml-ci
pipeline name: Templates - AWS CI 3.0
pipeline url:  https://dev.azure.com/VisualLansa/Lansa%20AWS%20Scalable%20License%20Images/_build?definitionId=54

### CODE REVIEW

File: aws-yaml-ci.yml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  value: $(Build.Repository.LocalPath)\cookbooks
+- name: CookbooksSource1
+  value: c:\lansa
+- name: regionName
+  value: ap-southeast-2
+- group: "Default Branch"
+- group: "Template Parameters"
+- group: "VPC ap-southeast-2"
+- group: "ImageVersion"
+- group: "ImageVersion - New"
```
Comment (robe070) [inline aws-yaml-ci.yml]: Why has this new variable group been created? Put the new 2022 variables in the ImageVersion variable group. Delete "ImageVersion - New"

File: aws-yaml-ci.yml [hunk 2]
```diff
@@ -0,0 +1,177 @@
+# Variable 'AWS_ACCESS_KEY_ID' defined in the Variable Group tab
```
Comment (robe070) [inline aws-yaml-ci.yml]: What purpose do these comments serve?

**Review by robe070 — COMMENTED** [review]
The PR comment needs to use the full reference to the Issue robe070/cookbooks#184

File: aws-yaml-ci.yml (hunk 1 above)
Comment (hirendra-singh) [reply aws-yaml-ci.yml]: Variable group 'ImageVersion - New' deleted and windows 2022 values added in 'ImageVersion' group.

File: aws-yaml-ci.yml (hunk 2 above)
Comment (hirendra-singh) [reply aws-yaml-ci.yml]: these comments were auto populated when yaml file exported from classic pipeline. deleted those comments.

Comment (robe070) [reply aws-yaml-ci.yml]: I know that. The question is why have you left them in! What is your purpose in leaving them in?

**Pull request merged**